docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML authentication in UI
+title: Configure SAML authentication in Grafana's UI
+weight: 1150
+---
+
+# Configure SAML authentication in Grafana
```

Comment:
```suggestion
# Configure SAML authentication using the Grafana user interface
```

Reply:
Title in the front matter and the H1 need to match. We can go longer with that title and use menutitle to control how the topic title appears in the TOC.